index.bs

```diff
@@ -954,17 +883,72 @@ interface FileReader: EventTarget {
   attribute EventHandler onabort;
   attribute EventHandler onerror;
   attribute EventHandler onloadend;
-
 };
 </pre>
 
+<div algorithm="read operation">
+A {{FileReader}} has an associated <dfn id=readOperation>read operation</dfn> algorithm,
+given |blob|, a |type| and an optional |encodingName|,
+runs the following steps:
+
+1. Set the [=context object=]'s {{FileReader/result}} to `null`.
```

Comment:
Maybe not something to clean up now, but I'd rather say "A FileReader _fr_ has an associated..." and then use _fr_ instead of the context object throughout.